src/examples/payment/createIndustryData.ts

```diff
@@ -0,0 +1,134 @@
+/**
+ * Mercado Pago Payment Capture.
+ *
+ * @see {@link https://www.mercadopago.com/developers/en/reference/payments/_payments/post Documentation }.
+ */
+
+import MercadoPago, { Payment } from '../../index';
+
+const client = new MercadoPago({
+	accessToken: '{{ACCESS_TOKEN}}',
+	options: { timeout: 5000 },
+});
+
+const payment = new Payment(client);
+
+payment
+	.create({
+		body: {
+			transaction_amount: 120.34,
+			description: 'Título do produto',
```

Comment:
[req] Deixar os textos em inglês
```suggestion
			description: 'Product title',
```